okcurl/build.gradle

```diff
@@ -1,3 +1,9 @@
+plugins {
+  id "com.palantir.graal" version "0.7.1-20-g113a84d"
+}
+
+import org.apache.tools.ant.taskdefs.condition.Os
```

Comment:
ant?!

Reply:
It's apparently still the canonical way I think.

Reply:
https://stackoverflow.com/questions/11235614/how-to-detect-the-current-os-from-gradle